server/channels/api4/license.go

```diff
@@ -267,7 +267,7 @@ func requestRenewalLink(c *Context, w http.ResponseWriter, r *http.Request) {
 	// check if it is possible to renew license on the portal with generated token
 	status, e := c.App.Cloud().GetLicenseSelfServeStatus(c.AppContext.Session().UserId, token)
 	if e != nil {
```

Comment:
First time seeing a single letter error var name :)

Reply:
Error names in the Mattermost code base are special :star: